sdk/scheduler/src/test/java/com/mesosphere/sdk/specification/util/RLimitTest.java

```diff
@@ -0,0 +1,38 @@
+package com.mesosphere.sdk.specification.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RLimitTest {
+    @Test
+    public void testRLimitCreationSucceeds() throws RLimit.InvalidRLimitException {
+        RLimit rlimit = new RLimit("RLMT_AS", 0L, 1L);
+
+        Assert.assertEquals(rlimit.getName(), "RLMT_AS");
+        Assert.assertEquals(rlimit.getSoft().get(), Long.valueOf(0));
+        Assert.assertEquals(rlimit.getHard().get(), Long.valueOf(1));
+    }
+
+    @Test
+    public void testRLimitCreationSucceedsWithUnlimitedLimits() throws RLimit.InvalidRLimitException {
+        RLimit rlimit = new RLimit("RLMT_AS", -1L, -1L);
+
+        Assert.assertEquals(rlimit.getSoft().get(), Long.valueOf(-1));
+        Assert.assertEquals(rlimit.getHard().get(), Long.valueOf(-1));
+    }
+
+    @Test(expected = RLimit.InvalidRLimitException.class)
+    public void testRLimitRequiresValidName() throws RLimit.InvalidRLimitException {
+        new RLimit("NONSENSE", 0L, 1L);
+    }
+
+    @Test(expected = RLimit.InvalidRLimitException.class)
+    public void testRLimitRequiresBothLimits() throws RLimit.InvalidRLimitException {
+        new RLimit("NONSENSE", 0L, -1L);
```

Comment:
Should have a valid name to ensure that the bad limit is what's throwing the exception? (here and below)